applovin_max/ios/Classes/AppLovinMAX.m

```diff
@@ -1257,6 +1222,50 @@ + (CGSize)adViewSizeForAdFormat:(MAAdFormat *)adFormat
              @"dspName" : ad.DSPName ?: @""};
 }
 
++ (ALGender)getAppLovinGender:(nullable NSString *)gender
```

Comment:
The Unity plugin for this method is written in C++, but since this is ObjC, `get` prefix for getters is not a thing, so this would have to be renamed to `toAppLovinGender:` or something.

Reply:
Understood.   Renamed to `toAppLovinGender` and `toAppLovinAdContentRating`.